doc/ctc-meetings/2016-01-27.md

```diff
@@ -0,0 +1,295 @@
+
+### CTC Membership Nominations [#4750](https://github.com/nodejs/node/issues/4750)
+
+Jeremiah: We’re going to keep this open for a couple weeks while the nominees sit in. The vote is TBD. We don’t do this often so the process is rusty.
+
+### buffer: add Buffer.from(), Buffer.alloc() and Buffer.allocUnsafe(), soft-deprecate Buffer(num) [#4682](https://github.com/nodejs/node/pull/4682)
+
+James: Where we’re at right now: continuing to update the PR and the EPS proposal that goes along with this based on the conversation. The new APIs seem uncontroversial. Unsurprisingly the name of one method is the most controversial. Currently, it’s allocUnsafe, a few folks don’t like “Unsafe”, would prefer “Raw”. [CD - yay bikesheds!]
+
+Some of the feedback from Trevor was that the API should be tweaked a bit, where we can pass encoding or a buffer to `.fill`, other than that the conversation around it is dying down. We haven’t decided if we’re going to change the default `new Buffer(n)` to zero-fill — if we do make that change we have to make it all the way down to v0.10 to keep it consistent.
+
+Mikeal: Call it buffer.remoteDisclosure().
+
+Just kidding.
+
+Ben: if we could make zero-fill as fast as the current impl, then we would of course pick that.
+
+James: Yes.
+
+Ben: I’ve been playing with doing init in a separate thread. Perhaps I can close the gap.
+
+Mikeal: The only objection is perf.
+
+James: Yep. The baseline impl is that zero fill uses calloc.
+
+Trevor: We use a pool. It’s always been possible to reach the pool from any buffer instance. Do we want to remove pooling?
+
+James: Current impl is that zerofill does not pool.
+
+Trevor: So much overhead!
+
+_(in document chat)_
+
+_Chris: why not use WeakMap for pooling?_
+
+_Ben: no weakmaps in v0.10._
+
+Jeremiah: Do we have benchmarks for calloc?
+
+James: Once we get a little further, I’ll revisit benchmarks.
+
+Trevor: Can the kernel do magic where memory isn’t actually allocated when it’s done in a tight loop? [CD - I didn’t capture this completely]
+
+Ben: You can mmap memory, memory isn’t actually allocated + zerofilled until you touch it, but touching it causes a page fault.
+
+### Buffer(number) is unsafe [#4660](https://github.com/nodejs/node/issues/4660)
+
+Jeremiah: Looks like we covered this.
+
+### doc: add Docs working group [#4244](https://github.com/nodejs/node/pull/4244)
+
+Chris: Onboarded some folks. Completed drafting the docs roadmap proposal. Making sure we have adequate tooling to build docs to the website from the node.js core repo.
+
+Chris: There is a docs wg meeting next week [10AM PST] [Wednesday @ 10am Feb 3rd]
+
+Chris: Also working with the inclusivity WG to make a new collaborator guide.
+
+Ben: No loosening of commit formatting, right?
+
+Chris, Mikeal: nope
+
+Jeremiah: Has anyone read through the proposed charter?
+
+Mikeal: hasn’t changed recently
+
+Ben, James: read though it recently, lgtm
```

Comment:
:-) :-) Sure, I can point them out whenever I find :-) But then we can leave the chat transcripts as they are right? It would give the readers a feel that they are really dealing with humans, who can make mistakes just like them :D